fundamentos/bloco-03-introducao-a-html-e-css/dia-01-html-e-css-estruturas-de-pagina/index.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+  <head>
+    <meta charset="UTF-8" />
+    <title>HTML</title>
+  </head>
+  <body>
+    <h1>Exercícios</h1>
+    <h2>Lista de pessoas que já fiz amizade na Trybe:</h2>
+    <ul>
+      <li><a href="#ronaldo">Ronaldo</a></li>
+      <li><a href="#danilo">Danilo</a></li>
```

Comment:
Esse nome aqui deixou a lista mais TOP kkkk

```suggestion
      <li><a href="#danilo">Danilo TOP</a></li>
```